_sections/visibility-android-studio.md

```diff
@@ -62,6 +62,12 @@ The white dot next to the file name indicates that your teammate is active in th
 
 ![See which file your teammates are on](/uploads/android-studio-visibility-file.jpeg "Which File"){:class="screenshot"}
 
+### Which issue are they currently working on?
+
+Once an issue tracker has been connected, GitLive's [issue tracking](/docs/issuetracking/){:class="internal-link"} feature can be used. This allows you to pinpoint which exact issues your teammates are currently making progress on. This can be used to give teammates more context than a simple branch name.
```

Comment:
This allows you to share the current issue you are working on, providing your teammates with more context than the repository / branch level.